.github/actions/build-frontend/action.yml

```diff
@@ -0,0 +1,41 @@
+# action.yml
+name: 'Build Frontend'
+description: 'Build react website'
+
+runs:
+  using: "composite"
```

Comment:
curious what this is?

Reply:
it's basically a multi-action ...action, instead of a docker or javascript based action:
https://docs.github.com/en/actions/creating-actions/about-custom-actions#types-of-actions

Reply:
neat